brunch-config.js

```diff
@@ -1,64 +1,64 @@
-var neat = require("bourbon-neat").includePaths
-var bourbon = require("bourbon").includePaths
+var neat = require('bourbon-neat').includePaths
+var bourbon = require('bourbon').includePaths
```

Comment:
Possible undefined variable: 'require' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)